CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-Remove mongodb dependence from packages.json (already in iota-node-lib)
+- Update npm shrinwrap file
```

Comment:
I think updates of the shrinkwrap file are considered minor and not included in CNR. Note that in the "parallel" PR of https://github.com/telefonicaid/iotagent-json/pull/278 we havent done such change.